### DISCUSSION TITLE
Server Timeout when using "next-translate"

### DISCUSSION THREAD

**Discussion by kxzo** [vercel/community#925] [Help] [answered]
I never got the server timeout on my vercel website after deploying but since I deployed a version with "next-translate" I sometimes get for some specific pages this error.

So, I would love hearing possible solutions for this issue, triggered by "next-translate" (and probably by a gSSP in the "next-translate" module).

By the way, in all my code I use swr instead of server side props, so this issue is truly triggered by "next-translate".

**Comment by amyegan** [comment, marked as answer]
Hi @einKnuffy! It sounds like you have traced the error to `next-translate`, so I think it would be best to reach out to [the next-translate community](https://github.com/aralroca/next-translate/discussions) for help specific to that library. They will have more specific knowledge about the package, how it should work, and why you're seeing unexpected behavior.

For the fastest help, be sure to provide a minimal reproduction and a description of what actions are supposed to happen in your code before the error occurs.

**Comment by kxzo** [comment]
@amyegan
That's right. But in fact this is a problem with Vercel too. When running a local server without the limitation of time of running tasks everything works. It's actually just a problem when deployed because the taks need to much time that I can't afford (through the free plan).
You can see this error on: https://shiromeet.vercel.app/de/game/fall-guys .
Although: https://shiromeet.vercel.app/de/home works fine
